frontends/bmd/src/app_root.tsx

```diff
@@ -750,6 +782,11 @@ export function AppRoot({
         logger={logger}
         unconfigureMachine={unconfigure}
         isMachineConfigured={Boolean(optionalElectionDefinition)}
+        resetPollsToPaused={
+          pollsState === 'polls_closed_final'
+            ? makeAsync(resetPollsToPaused)
```

Comment:
I created this `makeAsync` utility with some very fancy typescript, but I wonder if there is a built-in solution to this problem.

Reply:
Ah, interesting - yeah, I can't think of any alternative solutions except maybe defining the original function as `async` to begin with:

```
useCallback(async () => await Promise.resolve(dispatchAppState(/* etc */))
```

but i kinda like the idea of having the `makeAsync` around to make things more explicit